services/QuillLMS/app/helpers/navigation_helper.rb

```diff
@@ -18,9 +18,9 @@ module NavigationHelper
   LOG_IN = 'Log In'
   LOG_OUT = 'Log Out'
   MY_ACCOUNT = 'My Account'
-  MY_ACTIVITIES = 'My Activities'
-  MY_CLASSES = 'My Classes'
-  MY_REPORTS = 'My Reports'
+  MY_ACTIVITIES = 'Manage Activities'
+  MY_CLASSES = 'Manage Classes'
+  MY_REPORTS = 'View Reports'
```

Comment:
we should change these const names to match the strings

also, in this same file, we should update the definition names (i.e. `my_classes_tab`, etc)